tests/data/ericsson-6600_6600.json

```diff
@@ -4,16 +4,16 @@
             "devices": [
```

Comment:
renaming ciena-saos_6500.json → tests/data/ericsson-6600_6600.json seems wrong?

Reply:
No, that is not correct... Restored ciena-saos_6500.json.